### DISCUSSION TITLE
Can't syncronize times with the app

### DISCUSSION THREAD

**Discussion by rototom** [kimai/kimai#2444] [Questions & Answers]
My kimai2 installation seems to be broken.
When I use the app "Kimai 2 offline tracker" the recorded times won't be syncronized to the server, but stay in the app as "Saved timesheet entries". I can't see any suspicious things in the logs files of kimai.
What can I do?

**Comment by kevinpapst** [comment]
Test Kimai API with Postman from the same network where the app is running

**Reply by rototom** [reply]
I just made a new test. If I try to syncronize times one by one, it works. If I "collect" saved times and try to sny them all at once, they seem to get stuck.

**Reply by kevinpapst** [reply]
Then discuss that with Johannes in the Gitlab issue tracker

**Reply by rototom** [reply]
Johannes' app runs fine with the demo creds, so I must be a problem with my kimai installation.

**Reply by rototom** [reply]
And, sorry, what is "postman" and how do I test something with it?

**Reply by kevinpapst** [reply]
https://letmegooglethat.com/?q=postman

**Reply by rototom** [reply]
Thank you. I now think I understand what postman is. I uploaded my doc.json and authenticated. What test should I do now?

**Reply by rototom** [reply]
After playing around a bit with postman, I get "Error: Header "X-AUTH-TOKEN" contains invalid characters" is this something worth checking?

**Reply by rototom** [reply]
I reinstalled kimai2 completely and it works now. When I import my old sql data, it does not work anymore. So there seems to be a problem with the sql data?
Can I export customers, projects and activities from the old install and import them to the new one?

**Reply by rototom** [reply]
I can't simply import the exported timesheets, because of validation errors:
    [ERROR] Unknown user Name Surname in row 1
